packages/scripts/src/assessCases.ts

```diff
@@ -0,0 +1,228 @@
+import { strict as assert } from "assert";
+import { normalizedSquareMagnitudeDifference } from "./squareMagnitude";
+import { calculateEmbeddings } from "./calculateEmbeddings";
+import { SimpleTextGenerator } from "./SimpleTextGenerator";
+import { Embedder } from "mongodb-rag-core";
+import {
+  LlmAsJudgment,
+  PromptAndEmbeddings,
+  Relevance,
+  RelevanceMetrics,
+  ScoredPromptAndEmbeddings,
+} from "./Case";
+import { cosineSimilarity } from "mongodb-rag-core/aiSdk";
+
+/**
+  Given the expected answer, generate a number of possible prompts that could
+  elicit that expected answer.
+ */
+export const generatePromptsFromExpectedAnswer = async ({
```

Comment:
would be good to have lite evals for this.

not sure what metric you'd use to grade outputs, but at least good to have a few test cases that you can run this over for if you change the prompt.